string                      id
mac_ros_bridge/Item[]       items
uint64                      deadline